.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="1.8" />
+    <bytecodeTargetLevel target="11" />
```

Comment:
I don't think it's a good idea.

Reply:
Idk why but it crashes for me without it. We will duscuss that next meeting nbut for the demo I need this